packages/jellyfish-transaction/__tests__/index.test.ts

```diff
@@ -46,3 +53,85 @@ it('should be able to use DeFiTransactionConstants constants to craft Transactio
   new CTransactionSegWit(data).toBuffer(buffer)
   expect(buffer.toBuffer().toString('hex')).toStrictEqual(hex)
 })
+
+it('should be able to parse a TestNet TxId: 68c7902823139c4f5e4e81993e366e3499c8c9a90b4e41dd1fc1290d3735b30d from height: 1077842', function () {
```

Comment:
yea just suggestion.. 
better adding the reference 
https://defiscan.live/transactions/68c7902823139c4f5e4e81993e366e3499c8c9a90b4e41dd1fc1290d3735b30d?network=TestNet
friendly for us to cross check the result

Reply:
Can help me create a comment suggestions